### ISSUE TITLE
Change the order of the indicator implementations fallback

### ISSUE THREAD

**Issue by pacho2** [slgobinath/SafeEyes#502]
Currently trayicon plugin will try to load old libappindicator implementation and, if not found, fallback to the new one:
```try:
    gi.require_version('AppIndicator3', '0.1')
    from gi.repository import AppIndicator3 as appindicator
except:
    #fall back to Ayatana
    gi.require_version('AyatanaAppIndicator3', '0.1')
    from gi.repository import AyatanaAppIndicator3 as appindicator
```

the problem is that, during the transition, it is common to have both installed (as they can be installed at the same time) and, then, it will keep using the old (unmaintained) version over the new one.

I would then reverse the logic to fallback to the old implementation if ayatana one is not present

Thanks a lot

**Comment by AdamPS** [comment]
Great it seems that you know what the fix is. If you could create a pull request and test it works for you then we can merge it.

**Comment by archisman-panigrahi** [comment]
@deltragon Is this still relevant? I guess now we are only using org.kde.StatusNotifierItem

**Comment by deltragon** [comment]
Yeah, this has been fixed by #558.
